pkg/target/target.go

```diff
@@ -57,10 +62,11 @@ type Target interface {
 }
 
 type targetImpl struct {
-	Garden  string `yaml:"garden,omitempty" json:"garden,omitempty"`
-	Project string `yaml:"project,omitempty" json:"project,omitempty"`
-	Seed    string `yaml:"seed,omitempty" json:"seed,omitempty"`
-	Shoot   string `yaml:"shoot,omitempty" json:"shoot,omitempty"`
+	Garden           string `yaml:"garden,omitempty" json:"garden,omitempty"`
+	Project          string `yaml:"project,omitempty" json:"project,omitempty"`
+	Seed             string `yaml:"seed,omitempty" json:"seed,omitempty"`
+	Shoot            string `yaml:"shoot,omitempty" json:"shoot,omitempty"`
+	ControlPlaneFlag bool   `yaml:"controlPlane,omitempty" json:"controlPlane,omitempty"`
```

Comment:
We could also do something like kubernetes does with TypeMeta:
```go
type shootTargetImpl {
	Name            string `json:"shoot,omitempty"`
	ControlPlane bool
}
type targetImpl struct {
	Garden           string 
	Project          string
	Seed             string
	Shoot        shootTargetImpl `json:",inline"`
}
```

Reply:
Yes, this sounds reasonable. The control plane is always tied to the shoot. Do you think we should discuss this or do we just keep this in mind as an optimization for the future? Changing the target file spec should not have much impact on the user as it is mostly sed for one session only.